homeassistant/components/sensor/darksky.py

```diff
@@ -122,14 +122,14 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
 
     sensors = []
     for variable in config[CONF_MONITORED_CONDITIONS]:
-        sensors.append(ForeCastSensor(forecast_data, variable, name))
+        sensors.append(DarkSkySensor(forecast_data, variable, name))
 
     add_devices(sensors)
 
 
 # pylint: disable=too-few-public-methods
-class ForeCastSensor(Entity):
-    """Implementation of a Forecast.io sensor."""
+class DarkSkySensor(Entity):
```

Comment:
Maybe we should attribute the sensor data to Darksky? Like [yr.no](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/sensor/yr.py#L108)